backend/src/routes/blog.ts

```diff
@@ -204,13 +204,21 @@ blogRouter.get("/:id", async (c) => {
 
 blogRouter.delete("/:id", async (c) => {
   try {
-		const prisma = getDBInstance(c);
-    const postId = await c.req.param("id");
-    const post = await prisma.post.delete({
-      where: {
-        id: postId,
-      },
-    });
+    const postId = c.req.param("id");
+    const prisma = new PrismaClient({
+      datasourceUrl: c.env.DATABASE_URL,
+    }).$extends(withAccelerate());
+    
+    await prisma.tagsOnPost.deleteMany({
+      where:{
+        postId
+      }
+    })
+
+    await prisma.post.delete({
+      where: {id: postId}
+    })
+
```

Comment:
Nicely done. 

A couple of things here.
- Database consistency is important and could be lost IF the delete ON tagsOnPost executes but delete ON post fails.
- When we perform multiple db updates back-to-back in the same endpoint, it is generally a good practice to use 'Transactions'. 

OR

There is a more robust way to achieve this. That is, using `delete cascading`
Please read this: https://www.prisma.io/docs/orm/prisma-schema/data-model/relations/referential-actions

I think, we should go with delete cascading as it will take care of everything discussed internally.

Reply:
OK. On it.

Reply:
Enabled delete cascading.